Cython/Includes/numpy/__init__.pxd

```diff
@@ -973,22 +974,12 @@ cdef extern from "numpy/ufuncobject.h":
 
     int _import_umath() except -1
 
+cdef inline void set_array_base(object arr, object base):
+    Py_INCREF(base)
+    PyArray_SetBaseObject(arr, base)
```

Comment:
Just keep `ndarray arr` in the function signature of `set_array_base()`. That's what the signature was, and that's also what's needed for this call.

Reply:
ok